src/content/docs/fr/2/guide/upgrade-migrate/from-tauri-1.mdx

```diff
@@ -0,0 +1,1148 @@
+---
+title: Mise à Niveau Depuis Tauri 1.0
+
+i18nReady: true
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+
+Ce guide vous explique comment mettre à niveau votre application basée sur Tauri 1.0 vers Tauri 2.0.
+
+## Migration Automatisée
+
+
+Tauri v2 contient une commande `migrate` qui automatise la plupart des processus et vous aide à finir votre migration:
+
+<CommandTabs
+	npm="npm install @tauri-apps/cli@latest
+    npm run tauri migrate"
+	yarn="yarn upgrade @tauri-apps/cli --latest
+    yarn tauri migrate"
+	pnpm="pnpm update @tauri-apps/cli --latest
+    pnpm tauri migrate"
+	cargo='cargo install tauri-cli version "^2.0.0-alpha"
+    cargo tauri migrate'
+/>
+
+{/* TODO: 2.0 */}
+
+Apprenez-en plus à-propos de la commande `migrate` dans l'[interface de référence des commandes](/2/reference/cli#migrate) 

```

Comment:
Not sure about `interface de référence des commandes`

Would not be easier to say `guide des commandes` ou `manuel des commandes`

Reply:
I wasn't able to remember me the word during the translation, but i think I have even better : what about "Répertoire" ?